### PR TITLE
Pass along SignRequest

### PR DESCRIPTION
**Pull request by FredrikAreschoug**

### CODE REVIEW

**Comment by johantjellden** [comment]
Is member-service calling underrwriter on the MemberSignedEvent? I thought that it would call product-pricing who whould in turn call member-service.

**Comment by FredrikAreschoug** [comment]
Member-service is calling product-pricing if product-pricing doesn't find a product it goes to underwriter which in the end signs a product in underwriter

**Comment by johantjellden** [comment]
But underwriter also calls back to product-pricing?

**Comment by FredrikAreschoug** [comment]
Yes to create a signed product

File: src/main/kotlin/com/hedvig/underwriter/service/QuoteServiceImpl.kt [hunk 1]
```diff
@@ -240,34 +243,47 @@ class QuoteServiceImpl(
...
-
-        val signedProductId = quote.signedProductId
-            ?: if (!signedInMemberService) {
-                email?.let {
-                    productPricingService.createProduct(quote.getRapioQuoteRequestDto(it), quote.memberId).id
-                } ?: throw RuntimeException("No email when creating product")
-            } else {
-                productPricingService.createProduct(quote.createCalculateQuoteRequestDto(), quote.memberId).id
-            }
+        checkNotNull(quote.price) { "Quote must price to sign! Quote id: ${quote.id}" }
```
Comment (meletios) [inline src/main/kotlin/com/hedvig/underwriter/service/QuoteServiceImpl.kt]:
```suggestion
        checkNotNull(quote.price) { "Quote must have a price to sign! Quote id: ${quote.id}" }
```

**Pull request merged**